package.json

```diff
@@ -13,6 +13,7 @@
     "eslint-fix": "eslint nerdlets/ --fix"
   },
   "dependencies": {
+    "@newrelic/nr1-community": "^1.3.0-alpha.2",
```

Comment:
There's a newer alpha version available: [1.3.0-alpha.5](https://www.npmjs.com/package/@newrelic/nr1-community/v/1.3.0-alpha.5)

I believe it has the same API, but might have some bug fixes. 

CC: @devfreddy